command-line/lib/Command/Key/WalletId.hs

```diff
@@ -57,8 +59,8 @@ mod :: (Cmd -> parent) -> Mod CommandFields parent
 mod liftCmd = command "walletid" $
     info (helper <*> fmap liftCmd parser) $ mempty
         <> progDesc "Shows the cardano-wallet wallet ID for a given key"
-        <> footerDoc (Just $ string $ mconcat
-            [ "A wallet ID is a 40-digit hexadecimal string derived "
+        <> footerDoc (Just $ pretty $ mconcat
+            [ "A wallet ID is a 40-digit hexadecimal pretty derived " :: Text
```

Comment:
This looks like a situation where "find + replace" found a legitimate use of the word "string" and replaced it with pretty even though that's not desired behavior! :D

```suggestion
            [ "A wallet ID is a 40-digit hexadecimal string derived " :: Text
```